branchpro/apps/_sliders.py

```diff
@@ -60,6 +62,8 @@ def add_slider(
                                         dtype=np.float64
                                         )
 
+        mark_list = np.round(mark_list, decimals=2)
```

Comment:
```suggestion
        mark_list = np.round(mark_list, decimals=0 if as_integer else 2)
```
(this will require a slight change to the test as well)